src/zenml/integrations/aws/step_operators/sagemaker_step_operator.py

```diff
@@ -159,20 +159,36 @@ def launch(
 
         settings = cast(SagemakerStepOperatorSettings, self.get_settings(info))
 
-        session = sagemaker.Session(default_bucket=self.config.bucket)
-        instance_type = settings.instance_type or "ml.m5.large"
+        # Get and default fill SageMaker estimator arguments 
+        estimator_args = settings.estimator_args
+        session = estimator_args.sagemaker_session or sagemaker.Session(default_bucket=self.config.bucket)
+        instance_type = estimator_args.instance_type or "ml.m5.large"
```

Comment:
```suggestion
        instance_type = estimator_args["instance_type"] or "ml.m5.large"
```

This is a dict and needs to be accessed like this instead of `.attribute_name`, right? Same for the other lines